Pipfile

```diff
@@ -10,7 +10,7 @@ pyjwt = "*"
 requests = "*"
 
 [dev-packages]
-black = "==21.8b0"
+black = "==22.3.0"
```

Comment:
I think we should unpin the version for `black`

Reply:
Unpinned and added details for why black needed updating to the description